tests/ledctl/slot_test.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (C) 2023 Red Hat Inc.
+
+# Simple tests for ledctl.  This needs to be expanded.
+
+import pytest
+import logging
+from ledctl_cmd import LedctlCmd, Slot
+
+LOGGER = logging.getLogger(__name__)
+
+def test_non_slot_set_path(ledctl_binary, slot_filters, controller_filters):
+    """
+    Test setting the led status by using non-slot syntax, eg. ledctl locate=/dev/sda
+    """
+
+    cmd = LedctlCmd(ledctl_binary, slot_filters, controller_filters)
+    slots_with_device_nodes = [s for s in cmd.get_all_slots() if s.device_node is not None]
+
+    if len(slots_with_device_nodes) == 0:
+        pytest.skip("This test requires slots with devices but none found.")
+
+    for slot in slots_with_device_nodes:
+        for state in ["failure", "locate", "normal"]:
+            cmd.set_ibpi(slot.device_node, state)
+            cur = cmd.get_slot(slot, slot_filters)
+            assert cur.state == state,\
+                f"unable to set from {slot} to {state}, current = {cur} using non-slot syntax"
```

Comment:
Please add quotation marks next to variables.
Current example output:
AssertionError: unable to set from slot: 5 state: normal device: /dev/nvme0n1 to failure, current = slot: 5 state: normal device: /dev/nvme0n1 using non-slot syntax